x-pack/plugins/ingest_manager/server/services/epm/packages/assets.test.ts

```diff
@@ -55,10 +60,10 @@ const tests = [
   },
 ];
 
-test('testGetAssets', () => {
+test.skip('testGetAssets', () => {
```

Comment:
Mind adding a note about why this is skipped or when to re-enable it?

Reply:
This shouldn't be skipped, will fix. Thanks for spotting it!